src/test/java/com/amazon/dlic/auth/ldap/LdapBackendTestClientCert.java

```diff
@@ -152,8 +152,8 @@ public void testLdapSslAuth() throws Exception {
                 .putList(ConfigConstants.LDAP_HOSTS, "localhost:636")
                 .put(ConfigConstants.LDAP_AUTHC_USERSEARCH, "(uid={0})")
                 .put(ConfigConstants.LDAPS_ENABLE_SSL, true)
-                .put("opendistro_security.ssl.transport.keystore_filepath", "/Users/temp/opendistro_security_integration_tests/ldap/ssl-root-ca/spock-keystore.jks")
-                .put("opendistro_security.ssl.transport.truststore_filepath", "/Users/temp/opendistro_security_integration_tests/ldap/ssl-root-ca/truststore.jks")
+                .put("plugins.security.ssl.transport.keystore_filepath", "/Users/temp/opendistro_security_integration_tests/ldap/ssl-root-ca/spock-keystore.jks")
```

Comment:
use definition from `ConfigConstants`

Reply:
There are 505 results for all of the testing files, It would take significant amount of time to replace all the the strings to `ConfigConstants`.